hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmMultipartKeyInfo.java

```diff
@@ -257,10 +340,10 @@ public int hashCode() {
   }
 
   public OmMultipartKeyInfo copyObject() {
-    // For partKeyInfoList we can do shallow copy here, as the PartKeyInfo is
-    // immutable here.
+    // When PartKeyInfoMap is modified, a shallow copy will be created first,
+    // so here we can directly pass in partKeyInfoMap
```

Comment:
We should just mention that `PartKeyInfoMap` is immutable in the comment.

Reply:
done